django/forms/forms.py

```diff
@@ -189,6 +198,9 @@ def add_initial_prefix(self, field_name):
 
     def _html_output(self, normal_row, error_row, row_ender, help_text_html, errors_on_separate_row):
         "Output HTML. Used by as_table(), as_ul(), as_p()."
+        warnings.warn(
+            '`django.forms.BaseForm._html_output` is deprecated.'
+            ' Please use the `render` and `get_context` methods.', RemovedInNextVersionWarning)
```

Comment:
We wait two non-LTS versions in order to deprecate, no? So 5.0.`RemovedInNextVersionWarning` would be too soon.

Reply:
And, as you may know, deprecation entries needed in docs/internals/deprecation.txt and docs/releases/4.0.txt